dynamic_rest/fields/generic.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, embed=False, *args, **kwargs):
 
         super(DynamicGenericRelationField, self).__init__(*args, **kwargs)
         self.embed = embed
-        self.read_only = True
+        self.read_only = False
```

Comment:
I think we should just remove this. There may be cases where we'd want to set it on a per-field basis.